tasks/assert.yml

```diff
@@ -2,11 +2,21 @@
 - name: Check variables (RUN ONCE)
   assert:
     that:
-      - github_account is defined
```

Comment:
Why can't we just allow `github_account` to be set to the enterprise name in the enterprise case. That's what they do for the repository and organisation case as far as I can tell. Why do we need  to introduce the complexity of another variable? That's what my previous feedback was trying to point out.

Reply:
I'm not sure what you mean here @adamc00 is allowed just not required when its for the enterprise. 

Or is this to do with fetching the token?'

Reply:
Why create an additional variable when they are already using this one for the account or organisation name depending on the type of runner they are installing? We are just adding complexity by doing so as far as I can tell. Why not just allow the value of `github_account` to be the enterprise name when installing for an enterprise, following on from the pattern they already use.

This is based on the assumption that we don't need both values for some reason but we don't appear to.

Reply:
Yup, correct @adamc00  I misunderstood this.
Have fixed it up in the current commit, and tested.